.orchestra/support/revng-container/install.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+set -euo pipefail
+
+cd "$BUILD_DIR"
+UUID="$(uuidgen)"
+IMAGE_NAME="revng-container_$UUID"
+TEST_NAME=""
+
+function cleanup {
+  trap - EXIT;
+  if [ -n "${TEST_NAME-}" ]; then
+    podman kill "$TEST_NAME"
+    podman rm -f "$TEST_NAME"
+  fi
+  podman rmi -i "$IMAGE_NAME"
+}
+trap cleanup EXIT
+
+
+rsync \
+  --archive \
+  --verbose \
+  --progress \
+  "$ORCHESTRA_ROOT/revng" ./
+
+cp -a \
+  "$ORCHESTRA_DOTDIR/support/revng-container/"{apt.no-extra,entrypoint.sh,requirements.txt} \
+  ./
+
+podman build \
+  -t "$IMAGE_NAME" \
+  -f "$ORCHESTRA_DOTDIR/support/revng-container/Dockerfile" \
+  .
+
+podman save "$IMAGE_NAME" | xz -cv3 -T0 > "${TMP_ROOT}${ORCHESTRA_ROOT}/share/revng-container.tar.xz"
+
+if [ "$RUN_TESTS" -eq 1 ]; then
+  TEST_NAME="revng-container-test_$UUID"
+  TEST_PORT="$((1024 + RANDOM))"
+  podman run -d \
+    --name="$TEST_NAME" \
+    -p "127.0.0.10:$TEST_PORT:8000" \
+    --security-opt seccomp=unconfined \
```

Comment:
Why?

Reply:
https://medium.com/nttlabs/ubuntu-21-10-and-fedora-35-do-not-work-on-docker-20-10-9-1cd439d9921
This article is about docker, but podman seems to have a similar problem where the seccomp profile is blocking syscalls (in my case it shows up as "can't start new thread")

Reply:
Seems like `clone3` is a new syscall that's not listed among those whitelisted by default in older version of podman.
Where are you experiencing this? It seems more a configuration problem on the host machine.

Reply:
Ok, my bad, it was a stale file on my system. Sorry for the noise.